### DISCUSSION TITLE
미번역 사항 토론방

### DISCUSSION THREAD

**Discussion by NamGyu-Lee** [RimWorldKorea/RMK#6] [일반]
번역 안된 사항 확인

*Alpha Biomes* 
 - **checked**
 - https://steamcommunity.com/sharedfiles/filedetails/?id=1841354677%201841354677Alpha%20Biomes

---

*Dubs Rimatomics*
 - **checked**
 - https://steamcommunity.com/sharedfiles/filedetails/?id=1127530465%201127530465Rimatomics

---

*Vanilla Books Expanded*
 - **checked**
 - https://steamcommunity.com/sharedfiles/filedetails/?id=2193152410

---

*Xeva Faction*
 - https://steamcommunity.com/sharedfiles/filedetails/?id=2008120800

---

*Vanilla Cooking Expanded - Sushi*
 - **checked**
 - https://steamcommunity.com/sharedfiles/filedetails/?id=2158539170

---

*Vanilla Factions Expanded - Mechanoids*
 - **checked**
 - https://steamcommunity.com/sharedfiles/filedetails/?id=2329011599

---

*A RimWorld of Magic()*
- https://steamcommunity.com/sharedfiles/filedetails/?id=1201382956

**Comment by RimWorldKorea** [comment]
#7

**Reply by NamGyu-Lee** [reply]
현재 시간대로 해소된것들은 체크라고 표시해놨어요. 고마워요.

**Reply by RimWorldKorea** [reply]
#28 #29
Xeva Faction이랑 A RimWorld of Magic은 최신버전 RMK에서도 여전히 번역이 안되나요?
